test/processors/JsonSchemaInputProcessor.spec.ts

```diff
@@ -2,12 +2,27 @@ import * as fs from 'fs';
 import * as path from 'path';
 import { JsonSchemaInputProcessor } from '../../src/processors/JsonSchemaInputProcessor';
 import { CommonModel, Schema } from '../../src/models';
-import { simplify } from '../../src/simplification/Simplifier';
-jest.mock('../../src/simplification/Simplifier');
+import { Logger } from '../../src/utils';
+jest.mock('../../src/interpreter/Interpreter');
+jest.mock('../../src/utils/LoggingInterface');
 jest.spyOn(JsonSchemaInputProcessor, 'convertSchemaToCommonModel');
+
+let mockedReturnModels = [new CommonModel()];
```

Comment:
I should blame you every single time you use `let` or mutate something in your code ;)

Reply:
What do you mean 😄 ? 

This is the only way I found that could alter the mocked return, damn Jest..

Reply:
@jonaslagoni I mean that you should write inmutable code.